plat/allwinner/sun50i_a64/sunxi_power.c

```diff
@@ -13,6 +13,7 @@
 #include <platform_def.h>
 #include <sunxi_def.h>
 #include <sunxi_mmap.h>
+#include <sunxi_helpers.h>
```

Comment:
Hmm, this file should already be including `sunxi_private.h` for `sunxi_power_down`.